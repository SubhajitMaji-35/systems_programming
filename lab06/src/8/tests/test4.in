7  14 21
6 12 18
5 10 15
-1  0  1
-3 -2 -1
3  4  5
