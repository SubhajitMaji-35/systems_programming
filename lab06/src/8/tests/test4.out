14 12 10 
56 48 40 
98 84 70 
